File: SomeEasyDemos/代码规范/代码规范/ViewController.m
//  ViewController.m
//  代码规范
//
//  Created by 启业云 on 2019/8/5.
//  Copyright © 2019 启业云. All rights reserved.
//

#import "ViewController.h"
// Controlelr
#import "QYCLoginVC.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    
}

-(void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
    QYCLoginVC *loginVC = [[QYCLoginVC alloc] init];
    [self presentViewController:loginVC animated:YES completion:nil];
}

@end

File: 08_Swift/SwiftProject/AppCenter/AppCenter/ViewController.swift
//
//  ViewController.swift
//  AppCenter
//
//  Created by linxiang on 2023/5/13.
//

import UIKit

class ViewController: UIViewController {
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }
    
    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {}
    
    @IBAction func btnClick(_ sender: Any) {
        let a = AppCenterVC()
        self.navigationController?.pushViewController(a, animated: true)
    }
}
